labelbox/client.py

```diff
@@ -1597,3 +1597,49 @@ def get_model_slice(self, slice_id) -> ModelSlice:
         """
         res = self.execute(query_str, {"id": slice_id})
         return Entity.ModelSlice(self, res["getSavedQuery"])
+
+    class RemoveFeatureFromOntologyResult:
+        archived: bool
+        deleted: bool
+
+    def remove_feature_schema_from_ontology(
+            self, ontology_id: str,
+            feature_schema_id: str) -> RemoveFeatureFromOntologyResult:
+        """
+        Removes a feature schema from an ontology.
+        If the feature schema is a root level node with associated labels, it will be archived.
+        If the feature schema is a nested node in the ontology and does not have associated labels, it will be deleted.
+        If the feature schema is a nested node in the ontology and has associated labels, it will not be deleted.
+
+        Args:
+            ontology_id (str): The ID of the ontology.
+            feature_schema_id (str): The ID of the feature schema.
+
+        Returns:
+            RemoveFeatureFromOntologyResult: The result of the feature schema removal.
+
+        Example:
+            >>> client.remove_feature_schema_from_ontology(<ontology_id>, <feature_schema_id>)
+        """
+        ontology_endpoint = self.rest_endpoint + "/ontologies/" + urllib.parse.quote(
+            ontology_id) + "/feature-schemas/" + urllib.parse.quote(
+                feature_schema_id)
+        response = requests.delete(
+            ontology_endpoint,
+            headers=self.headers,
+        )
+
+        if response.status_code == requests.codes.ok:
+            response_json = response.json()
+            if response_json['archived'] == True:
+                print(
```

Comment:
Can we use logger.info instead of print here?

Reply:
[Updated](https://github.com/Labelbox/labelbox-python/pull/958/commits/6c470dd62b0f502e81bbea9f3bc5f5be6458c6c4)